README.md

```diff
@@ -1,22 +1,91 @@
 [![Build Status](https://travis-ci.com/HumanCompatibleAI/adversarial-policies.svg?branch=master)](https://travis-ci.com/HumanCompatibleAI/adversarial-policies)
 
-Preliminary research investigating adversarial policies: given a victim policy 
-in a multi-agent system, find a policy which will break the victim.
+Codebase to train, evaluate and analyze adversarial policies: policies attacking a victim in
+a multi-agent system. See [paper](https://arxiv.org/abs/1905.10615) for more information.
 
-# Setup
+# Installation
 
-This codebase assumes Python 3.6. Install the requirements in 
-`requirements-build.txt` before those in `requirements.txt`.
-Anaconda users can install directly with `conda env create -f environment.yml`.
+The easiest way to install the code is to build the Docker image in the `Dockerfile`.
+This will install all necessary binary and Python dependencies.
```

Comment:
My understanding of the current Docker build is that it requires a publicly available URL containing a MuJoCo key; this is given as an error within ci/local_tests.sh, but might since we don't expect people to be using that script it be worth articulating here

Reply:
You're right, I've added this.